packages/flutter/lib/src/material/data_table.dart

```diff
@@ -357,6 +358,11 @@ class DataCell {
 /// [PaginatedDataTable] which automatically splits the data into
 /// multiple pages.
 ///
+/// ## Performance considerations when wrapping [DataTable] with [SingleChildScrollView]
+///
+/// Wrapping a [DataTable] with [SingleChildScrollView] is expensive as [SingleChildScrollView]
```

Comment:
When https://github.com/flutter/packages/pull/4536 lands, this could point to using `TableView` instead?

cc @Piinks

Reply:
Yeah probably. :)

I would recommend adding some reference here to other options, not including TableView since that has not landed yet. If someone should not use a SingleChildScrollView because the table is very large, we should recommend PaginatedDataTable to split it up, or using a CustomScrollView.

TableView may not be relevant if someone only cares about scrolling in one direction, but we can add a reference to it when it is available.